style/sass/v-panel-subheader.scss

```diff
@@ -9,3 +9,11 @@ h2 {
   margin: 0;
   padding: 0 16px;
 }
+
+.icon {
```

Comment:
Not sure if you have any oppinions about this? I could color the image directly and loose the original color (blue). This will either make the icon on the panel also become gray or I will have to duplicate the image. This way I reduce the number of icons we use.